upgrade/scripts/upgrade/util/update-customizations.sh

```diff
@@ -125,6 +125,11 @@ if [ "$(yq4 eval '.spec.kubernetes.services."image-verification-policy"' "${upgr
   ' "$c"
 fi
 
+# rack-resiliency did not have configurable customization prior to 1.7. Import rack-resiliency settings from upgrade customizations file during upgrade.
+if [ "$(yq4 eval '.spec.kubernetes.services[\"rack-resiliency\"]' "$c")" == "null" ]; then
```

Comment:
Is there a reason not to simplify this (which also would make it consistent with how the check on line 106 does it)?
```suggestion
if [ "$(yq4 eval '.spec.kubernetes.services.rack-resiliency' "$c")" == null ]; then
```